cjs/interface/attr.js

```diff
@@ -42,8 +43,11 @@ class Attr extends Node {
 
   toString() {
     const {name, [VALUE]: value} = this;
-    return emptyAttributes.has(name) && !value && ignoreCase(this) ?
-      name : `${name}="${value.replace(QUOTE, '&quot;')}"`;
+    if (!value) {
```

Comment:
this changes the outcome or checks too much for all attributes without a value ... having `emptyAttributes.has(name)` as first guard for the common HTML case is desired and XML attributes are covered by the `ignoreCase(this)` check ... I would put that guard back ahead of the rest to avoid perf regression (too bad I don't have automated checks for that).

Reply:
XML is the edge/non-common case, HTML is the primary use case to date so ... thanks!